App/views/id.hbs

```diff
@@ -0,0 +1,14 @@
+<h1>Azure AD</h1>
+<h3>ID Token</h3>
+<table>
+    <tbody>
+        {{#each idTokenClaims}}
+        <tr>
+            <td>{{@key}}</td>
+            <td>{{this}}</td>
+        </tr>
+        {{/each}}
+    </tbody>
+</table>
+<br>
+<a href="/">Go back</a>
```

Comment:
>href [](http://example.com/codeflow?start=3&length=4)

Drop a text like "Learn about claims in this ID token at [https://aka.ms/id-tokens](https://aka.ms/id-tokens)"

Reply:
done